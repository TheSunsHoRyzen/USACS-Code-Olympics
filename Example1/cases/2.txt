Alice Math 88 English 92
Bob Math 79 English 85
Carol Math 91 English 89