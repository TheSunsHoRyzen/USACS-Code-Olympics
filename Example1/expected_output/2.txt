Subject: Math
Average Grade: 86.0
Top Student(s): Carol (91)

Subject: English
Average Grade: 88.66666666666667
Top Student(s): Alice (92)

